go/core/flow.go

```diff
@@ -172,7 +172,7 @@ type flowState[I, O any] struct {
 	FlowName string `json:"name,omitempty"`
 	// start time in milliseconds since the epoch
 	StartTime tracing.Milliseconds `json:"startTime,omitempty"`
-	Input     I                  `json:"input,omitempty"`
+	Input     I                    `json:"input,omitempty"`
```

Comment:
Shouldn't `flowState` change to `In, Out` ?

Reply:
Done.